hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/fs/ozone/AbstractRootedOzoneFileSystemTest.java

```diff
@@ -279,6 +280,9 @@ void initClusterAndEnv() throws IOException, InterruptedException, TimeoutExcept
     userOfs = UGI_USER1.doAs(
         (PrivilegedExceptionAction<RootedOzoneFileSystem>)()
             -> (RootedOzoneFileSystem) FileSystem.get(conf));
+    userO3fs = UGI_USER1.doAs(
+        (PrivilegedExceptionAction<OzoneFileSystem>)()
+            -> (OzoneFileSystem) FileSystem.get(conf));
```

Comment:
How does this work?  I would expect `FileSystem.get(conf)` to return the same implementation (even same instance if cache is not disabled) for both calls.